dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -577,10 +579,9 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 							</Hide>
 						</GridItem>
 						<GridItem area="meta">
-							<Hide from="desktop">
-								<div css={sidePaddingDesktop}>
-									<ArticleMeta
-										branding={branding}
+							{isApps && (
+								<Hide from="desktop">
```

Comment:
If both Apps and Web hide from desktop, we could leave this component one level up.